sql/2025/page-weight/pages_using_javascript.sql

```diff
@@ -0,0 +1,39 @@
+WITH totals AS (
+  SELECT
+    date,
+    client,
+    is_root_page,
+    COUNT(0) AS num_pages
+  FROM
+    `httparchive.crawl.pages`
+  WHERE
+    date = '2025-07-01'
+  GROUP BY
+    date,
+    client,
+    is_root_page
+)
+
+SELECT
+  date,
+  client,
+  is_root_page,
+  COUNTIF(INT64(custom_metrics.other.almanac.scripts.total) > 0) AS pages_using_js,
```

Comment:
Is there any different between:

`INT64(custom_metrics.other.almanac.scripts.total`

Used here and:

`CAST(JSON_VALUE(custom_metrics.other.almanac.scripts.total) AS INT64)`

Used elsewhere in this PR? And should be settle on one style?

Also what happens with NULLs or other non-integers? Is `SAFE_CAST` safer (which then also decides which format to use!)?